barcode/ean.py

```diff
@@ -4,6 +4,7 @@
 """
 __docformat__ = "restructuredtext en"
 
+from functools import reduce
```

Comment:
Linting is failing because there should be a new line after this import (there's usually a new line between stdlib and other imports).

Reply:
Thanks for your feedback. 

I tried to fix the "lint" report by 40b42a86b0af22699ac95dd5fd8b1c1a4b04c881.without success.
Feel free to modify my PR